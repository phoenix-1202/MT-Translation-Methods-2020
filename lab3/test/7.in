funWithManyBracketsAndUsingTrue :: Int -> Int -> Int
funWithManyBracketsAndUsingTrue a b | (((((5))))) == 5 || ((((+ ((((a _)))+12) 5) > 0))) && true = 5