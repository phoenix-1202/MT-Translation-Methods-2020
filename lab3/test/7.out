#include <bits/stdc++.h>;
using namespace std;

int funWithManyBracketsAndUsingTrue(int arg1, int arg2) {
if ((5 == 5) || ((((a() + 12) + 5) > 0) && true)) {
return 5;
}

}

//EXIT